iamserver/iamserver.go

```diff
@@ -27,22 +28,21 @@ import (
 
 	"github.com/aoscloud/aos_common/aoserrors"
 	"github.com/aoscloud/aos_common/api/cloudprotocol"
-	pb "github.com/aoscloud/aos_common/api/iamanager/v2"
+	pb "github.com/aoscloud/aos_common/api/iamanager/v3"
 	"github.com/aoscloud/aos_common/utils/cryptutils"
 	"github.com/golang/protobuf/ptypes/empty"
 	log "github.com/sirupsen/logrus"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 
-	"github.com/aoscloud/aos_iamanager/certhandler"
 	"github.com/aoscloud/aos_iamanager/config"
 )
 
 /***********************************************************************************************************************
  * Consts
  **********************************************************************************************************************/
 
-const discEncryptyonType = "diskencryption"
+const discEncryptionType = "diskencryption"
 
 const iamAPIVersion = 3
```

Comment:
change api version to 4

Reply:
But why? It implements proto v3.

Reply:
we have separate api version numeration. It was done bo cloud team reuest